stable/kong/README.md

```diff
@@ -122,10 +122,61 @@ $ helm install stable/kong --name my-release \
   --set=image.tag=0.11.2,env.database=cassandra,cassandra.enabled=true
 ```
 
-Alternatively, a YAML file that specifies the values for the above parameters can be provided while installing the chart. For example,
+Alternatively, a YAML file that specifies the values for the above parameters
+can be provided while installing the chart. For example,
 
 ```console
 $ helm install stable/kong --name my-release -f values.yaml
 ```
 
 > **Tip**: You can use the default [values.yaml](values.yaml)
+
+### Kong Ingress Controller
+
+Kong Ingress Controller's primary purpose is to satisfy Ingress resources
+created in k8s.  It uses CRDs for more fine grained control over routing and
+for Kong specific configuration. To deploy the ingress controller together with
+kong run the following command:
+
+```bash
+helm install . \
```

Comment:
This seems like it's being used during development but end users shouldn't really need to do it.
Should it rather be `helm install stable/kong`?